src/open62541_compat.cpp

```diff
@@ -269,6 +268,8 @@ OpcUaType UaVariant::type() const
         return OpcUaType_String;
     else if(m_impl->type == &UA_TYPES[UA_TYPES_BOOLEAN])
     	return OpcUaType_Boolean;
+    else if(m_impl->type == &UA_TYPES[UA_TYPES_BYTESTRING])
+    	return OpcUaType_ByteString;
```

Comment:
I agree we should do something about it; not yet evident to me what would be the best thing. Reason: not sure how well a hash-map scales with a low-size collection like 20-30 items. Also, to be checked it OpcUaType_xxx is actially stored in UA_TYPES - if that's the case, we would have a simple and elegant solution. I'll look into this further.